python/storage-initializer/scripts/initializer-entrypoint

```diff
@@ -11,4 +12,7 @@ src_uri = sys.argv[1]
 dest_path = sys.argv[2]
 
 logging.info("Initializing, args: src_uri [%s] dest_path[ [%s]" % (src_uri, dest_path))
-kfserving.Storage.download(src_uri, dest_path)
\ No newline at end of file
+kfserving.Storage.download(src_uri, dest_path)
```

Comment:
We might need to download to a temporary directory and rename it to `dest_path`.
Otherwise we risk the serving task will get confused by a partial download...

Reply:
unfortunately, `dest_path` is a mount point and such a rename results in `device busy` errors. Please see https://github.com/mesosphere/kfserving/pull/1 for a somewhat improved approach to this problem